lib/model/store.dart

```diff
@@ -287,6 +287,12 @@ class PerAccountStore extends ChangeNotifier with StreamStore {
 
   final Map<int, User> users;
 
+  Map<int, Set<int>> get streamUserIds => {..._streamUserIds};
+  final Map<int, Set<int>> _streamUserIds = {};
+
+  Set<int> get dmUserIds => {..._dmUserIds};
+  Set<int> _dmUserIds = {};
```

Comment:
These fields should get dartdoc explaining what their intended meaning is. That's helpful for the reader looking either at the code that initializes and maintains them, or at the code that consumes them.

In particular it separates those two problems — one can look only at some code that consumes the field and work out whether the code is correct, given the field's documented meaning, and separately one can look only at some code that maintains the field and work out whether that code is correct.

Without documentation of the intended meaning, one has to look at both sides together, guess what the meaning might be, then try to validate that guess against both sides, and potentially repeat with an adjusted guess.